mat64/dense_arithmetic.go

```diff
@@ -228,15 +228,7 @@ func (m *Dense) Inverse(a Matrix) error {
 			m.Copy(a)
 		}
 	default:
-		if m != aU {
-			m.Copy(a)
-		} else if aTrans {
-			// m and a share data so Copy cannot be used directly.
-			tmp := getWorkspace(r, c, false)
-			tmp.Copy(a)
-			m.Copy(tmp)
-			putWorkspace(tmp)
-		}
+		m.Copy(a)
```

Comment:
What do you mean does it work for RawTriangular? The rules say that we won't check for shadowing unless "the Raw type matches that of the receiver", which it doesn't here.